src/app/core/setup/demo-assistant-button/demo-assistant-button.component.scss

```diff
@@ -0,0 +1,19 @@
+@use "variables/breakpoints";
+
+.demo-assistant-btn {
+  @media screen and (max-width: breakpoints.$sm) {
+    min-height: 56px;
+  }
+  min-height: 65px;
```

Comment:
is it possible to work with relative heights (100%?) here instead or was this the only way to get it to work?